src/NHibernate.Test/NHSpecificTest/GH1594/Fixture.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Linq;
+using System.Threading;
+using System.Transactions;
+using NUnit.Framework;
+
+namespace NHibernate.Test.NHSpecificTest.GH1594
+{
+	[TestFixture]
+	public class Fixture : BugTestCase
```

Comment:
Since it uses system transaction, the fixture must be disabled if the environment does not support them. Add:

```c#
protected override bool AppliesTo(ISessionFactoryImplementor factory) =>
	factory.ConnectionProvider.Driver.SupportsSystemTransactions;
```

(Since currently this is false for all .Net Core / .Standard drivers, the private field fix is in fact not really needed, but it does not harm to have it.)

Reply:
Done, thanks for describing.